docs/src/markdown/documentation/faq.md

```diff
@@ -9,14 +9,31 @@ navigation:
 # Frequently Asked Questions
 
 
+### Isn't this basically React/Elm?
+
+[React](https://reactjs.org/) and [the Elm architecture](https://guide.elm-lang.org/architecture/) were both strong influences for this library. However both those libraries are written for JavaScript. Workflows are written in and for both Kotlin and Swift, making use of features of those languages, and with usability from those languages as a major design goal. There are also a few architectural differences:
+
+|  | React | Elm | Workflow |
+|---|---|---|---|
+| **Modularity** | `Component` | TK | `Workflow` is analogous to React's `Component` |
```

Comment:
Is `TK` how we're saying `TBD` these days?

Reply:
https://en.wikipedia.org/wiki/To_come_(publishing)